SU-HSDO/suhumsci
# STN-150 : Implement Feedback from Sprints 3 & 4

# [STN-150](https://sparkbox.atlassian.net/browse/STN-150) READY FOR REVIEW

## Summary
This PR addresses feedback provided after review of [Econ QA](https://economics-stage.stanford.edu) environment after Sprints 3 and 4. Credentials for QA are stored in 1Password.

## Need Review By (Date)
2/7/2020

## Urgency
low / medium

## Steps to Test
Run `npm run test` and verify all tests pass.

Go to a basic page that has horizontal card views. Verify the following feedback has been addressed:
1. Horizontal News Card Feedback
    - [x] No changes have been made to the code base. Verify correct H2 and hover styles are visible when the view Field setting for `Content: Title` > `Style Settings` > `Add default classes` is unchecked.
    - [x] Verify vertical spacing between cards.
2. General Feedback
    * Category Styles
        - [x] Update the horizontal card view settings for the `Content: Categories` Field to use an ordered list instead of an unordered list. Verify the styles for both are the same.
        * Bryan has changes to the categories styles in [PR 466](https://github.com/SU-HSDO/suhumsci/pull/466). See his note in the first comment of [STN-150](https://sparkbox.atlassian.net/browse/STN-150)
    - [x] Verify hover area of content link arrow has been increased to 44px by 44px to meet [WCAG AAA](https://www.w3.org/WAI/WCAG21/Understanding/target-size.html) target size compliance
3. Open Questions
    * Concerns regarding 3 column layout are being addressed in [STN-155](https://sparkbox.atlassian.net/browse/STN-155) While reviewing the QA site, I noticed the title overflowing into the column gutter at at small column widths. Styles have been added to prevent the title from overflowing into the column gutter.
    - [x] Confirm H2 text matches [Figma](https://www.figma.com/file/toNZnFLadtpNpG5dPzNGWz/⚙%EF%B8%8F-Colorful-Theme---Working?node-id=2067%3A0) mockups. Note: I did not make any changes here. I will share this feedback with Merani & J so that they are aware.
4. Design Feedback
_This was not in the Jira card, but an additional request that surfaced during Lindsay & Merani's pairing session._
    * The page horizontal margin at mobile sizes screens has been increased to match the Figma design comps. When I making this change, I also nudged up the margin of small and medium screen sizes so allow the margin to scale nicely between screen sizes.
    <img width="256" alt="mobile margin 28px" src="https://user-images.githubusercontent.com/12678977/73949973-24212600-48c9-11ea-8223-35ce49017615.png"> _Screenshot of Figma mobile comp_


## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
